model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -175,6 +176,72 @@ message PTransform {
   DisplayData display_data = 6;
 }
 
+message StandardPTransformUrns {
```

Comment:
Separate primitives, deprecated primitives, standardized composites.

Primitives are: ParDo, Flatten, GroupByKey, Impulse, AssignWindows

Deprecated primitives are: Read, CreateView

There's some mystery surrounding the status of GBK / CoGBK

The rest are just composite that have well known URNs. They should live as annotations closer to their payloads, but not alongside the primitives.

Reply:
+1